src/process_binding.cpp

```diff
@@ -0,0 +1,923 @@
+#include "process_binding.hpp"
+/* #include "attributes.hpp" */
+/* #include "namespaces.hpp" */
+#include "pystring.h"
+/* #include "type.hpp" */
+#include "clang/AST/ASTContext.h"
+#include "clang/AST/Decl.h"
+#include "clang/AST/DeclCXX.h"
+#include "clang/AST/DeclTemplate.h"
+#include "clang/AST/Type.h"
+#include "clang/ASTMatchers/ASTMatchers.h"
+#include "clang/Basic/LLVM.h"
+#include "llvm/Support/Casting.h"
+#include <bits/stdint-uintn.h>
```

Comment:
```suggestion
#include <cstdint>
```
i think this might be more portable.
stdint-uintn isn't available on my mac, cstdint is.